sdk/eventhubs/azure-messaging-eventhubs-checkpointstore-blob/README.md

```diff
@@ -0,0 +1,128 @@
+### Adding the package to your product
+
+```xml
+<dependency>
+    <groupId>com.azure</groupId>
+    <artifactId>azure-messaging-eventhubs-checkpointstore-blob</artifactId>
+    <version>1.0.0-preview.1</version>
+</dependency>
+```
+
+## Key concepts
+
+## Examples
+- [Consume events from all Event Hub partitions][sample_event_processor]
+
+### Consume events using an Event Processor
+
+To consume events for all partitions of an Event Hub, you'll create an [`EventProcessor`][source_eventprocessor] for a
+specific consumer group. When an Event Hub is created, it provides a default consumer group that can be used to get
+started.
+
+The [`EventProcessor`][source_eventprocessor] will delegate processing of events to a
+[`PartitionProcessor`][source_partition_processor] implementation that you provide, allowing your application to focus on the
+business logic needed to provide value while the processor holds responsibility for managing the underlying consumer operations.
+
+In our example, we will focus on building the [`EventProcessor`][source_eventprocessor], use the 
+[`BlobPartitionManager`][source_inmemorypartitionmanager], and a `PartitionProcessor` implementation that logs
+events received and errors to console.
+
+```java
+class Program {
+    public static void main(String[] args) {
+        EventProcessor eventProcessor = new EventHubClientBuilder()
+            .connectionString("<< CONNECTION STRING FOR THE EVENT HUB INSTANCE >>")
+            .consumerGroupName("<< CONSUMER GROUP NAME>>")
+            .partitionProcessorFactory(SimplePartitionProcessor::new)
+            .partitionManager(new BlobPartitionManager(containerAsyncClient))
+            .buildEventProcessor();
+
+        // This will start the processor. It will start processing events from all partitions.
+        eventProcessor.start();
+
+        // When the user wishes to stop processing events, they can call `stop()`.
+        eventProcessor.stop();
+    }
+}
+
+class SimplePartitionProcessor extends PartitionProcessor {
+    @Override
+    Mono<Void> processEvent(PartitionContext partitionContext, EventData eventData) {
+        System.out.printf("Event received. Sequence number: %s%n.", eventData.sequenceNumber());
+        return partitionContext.updateCheckpoint(eventData);
+    }
+}
+```
+
+## Troubleshooting
+
+### Enable client logging
+
+You can set the `AZURE_LOG_LEVEL` environment variable to view logging statements made in the client library. For
+example, setting `AZURE_LOG_LEVEL=2` would show all informational, warning, and error log messages. The log levels can
+be found here: [log levels][source_loglevels].
+
+## Next steps
+
+## Contributing
+
+If you would like to become an active contributor to this project please refer to our [Contribution
+Guidelines](./CONTRIBUTING.md) for more information.
+
+<!-- Links -->
+[amqp_transport_error]: https://docs.oasis-open.org/amqp/core/v1.0/os/amqp-core-transport-v1.0-os.html#type-amqp-error
```

Comment:
Are all these links used?

Reply:
No, I am working on fixing the README for this. Will send out a PR for updated README separately.